pyvista/utilities/fileio.py

```diff
@@ -233,3 +249,106 @@ def read_exodus(filename,
 
     reader.Update()
     return pyvista.wrap(reader.GetOutput())
+
+
+def read_meshio(filename, file_format = None):
+    """Read any mesh file using meshio."""
+    from meshio._vtk import (
+        meshio_to_vtk_type,
+        vtk_type_to_numnodes,
+    )
+    
```

Comment:
Make sure relative paths like `"~/Desktop.foo.vti"` will work
```suggestion

    filename = os.path.abspath(os.path.expanduser(filename))
```